src/cuga/backend/cuga_graph/nodes/chat/chat_agent/mcp_filesystem_wrapper.py

```
@@ -0,0 +1,200 @@
+        description=tool.description or "",
+        args_schema=getattr(tool, "args_schema", None),
+    )
+    return wrapped
+
+
+def _skills_enabled() -> bool:
+    try:
+        from cuga.config import settings
+
+        return bool(getattr(getattr(settings, "skills", None), "enabled", False))
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)